README.md

```diff
@@ -462,35 +464,52 @@ The `uf2create` task is used to generated an uf2 binary suitable for running on
 
     Use this plugin to create Raspberry Pico uf2 files from an AtomVM packbeam file.
 
-    Usage: rebar3 atomvm uf2create [-o <output>] [-s <start>] [-i <input>]
+    Usage: rebar3 atomvm uf2create [-u <family_id>] [-o <output>]
+                               [-a <start>] [-i <input>]
 
-    -o, --output  Output path/name
-    -s, --start   Start address for the uf2 binary (default 0x10180000)
-    -i, --input   Input avm file to convert to uf2
+      -f, --family_id  Flavor of uf2 file to create (default rp2040)
```

Comment:
@pguyot, I wonder if the short option should be `-m` like picotool. The `-f` option seems to match “family” better, but might be ambiguous because this might be assumed to mean either “file” or “force”… what do you think?

Reply:
I wasn't aware picotool accepted `-m` shortcut as I used `--family-id`. Since it does, `-m` looks good.

Reply:
I must been up too late last night when I read that... `picotool` does not have a short option for `--family`, so I am going to keep this as `-f`.